.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,69 @@
+name: Bump Webhook RC against rancher/charts
+on:
+  workflow_dispatch:
+    inputs:
+      charts_ref:
+        description: "Submit PR against the following rancher/charts branch (eg: dev-v2.9)"
+        required: true
+        default: "dev-v2.9"
+      prev_webhook:
+        description: "Previous Webhook version (eg: 0.5.0-rc13)"
+        required: true
+        default: ""
+      new_webhook:
+        description: "New Webhook version (eg: 0.5.0-rc14)"
+        required: true
+        default: ""
+
+jobs:
+  create-rancher-charts-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{github.event.inputs.ref}}
+          path: webhook
+
+      # Checkout the fork first because it's simpler to setup the token for the
+      # push later on
+      - name: Checkout fork of rancher/charts
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          repository: tomleb/rancher-charts
+          token: ${{ secrets.PUSH_TO_FORKS_TOKEN }}
+          path: charts
+
+      - name: Install dependencies
+        run: sudo snap install yq --channel=v4/stable
+
+      - name: Setup repo dir
+        run: |
+          cd charts
+          BRANCH="gha-test-${GITHUB_RUN_ID}-${GITHUB_RUN_ATTEMPT}"
+          echo "BRANCH=${BRANCH}" >> $GITHUB_ENV
+          git config --global user.email "webhook@example.com"
+          git config --global user.name "tomleb webhook Bot"
+          git remote add upstream https://github.com/rancher/charts
+          git fetch upstream
+          git checkout -b $BRANCH upstream/${{github.event.inputs.charts_ref}}
+
+      - name: Run release script
+        run: |
+          ./webhook/.github/workflows/scripts/release-against-charts.sh charts ${{github.event.inputs.prev_webhook}} ${{github.event.inputs.new_webhook}}
```

Comment:
Feel free to correct me if I'm wrong, but
`prev_webhook` and `new_webhook` smell too much like user-supplied input and unless you know they can't contain semi-colons and other control characters, should probably be passed to `release-against-charts.sh` as environment vars

Reply:
Updated to use environment variables instead, let me know what you think!